charts/devx-dashboard/Chart.yaml

```diff
@@ -1,20 +1,16 @@
 apiVersion: v2
 name: devx-dashboard
+
 description: A developer dashboard for releasing software
-version: 1.1.3
-appVersion: 1.0.6
+version: 2.0.0-rc1
+appVersion: 2.0.8
 dependencies:
 - name: iam-service-account
-  version: 1.1.4
+  version: 2.0.0
   repository: https://ok-amba.github.io/config-connector-helm
   alias: iamServiceAccount
   condition: deadLetter.enabled
 - name: sqlinstance
-  version: 2.1.4
+  version: 3.0
   repository: https://ok-amba.github.io/config-connector-helm
   condition: deadLetter.enabled
-- name: db-migration
```

Comment:
Why has this been deleted?

Reply:
I see it was added  in one of the templates.

Reply:
Yes, is was integrated to make the chart less complex.